### DISCUSSION TITLE
Could not resolve dependency: peer wretch@"^1.7.10" from @withtally/tally-publish-dao@0.0.5

### DISCUSSION THREAD

**Discussion by apoorvam-web3** [reserve-protocol/protocol#505] [Q&A] [answered]
When I do npm install using node v16.14.0 (npm v8.3.1)

I get this error

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: reserve-protocol@1.0.0
npm ERR! Found: wretch@2.1.5
npm ERR! node_modules/wretch
npm ERR!   dev wretch@"^2.0.4" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer wretch@"^1.7.10" from @withtally/tally-publish-dao@0.0.5
npm ERR! node_modules/@withtally/tally-publish-dao
npm ERR!   dev @withtally/tally-publish-dao@"^0.0.5" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/apoorvam/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/apoorvam/.npm/_logs/2022-12-02T10_35_03_008Z-debug-0.log

**Comment by julianmrodri** [comment, marked as answer]
We use `yarn` for resolving dependencies using `yarn.lock`.  Do a cleanup with `npx hardhat clean`, then remove the `node_modules` folder. Make sure you have the lates versiont checked out and do `yarn install`.
